exonum/src/runtime/mod.rs

```diff
@@ -191,6 +191,27 @@ impl From<RuntimeIdentifier> for u32 {
     }
 }
 
+impl std::convert::TryFrom<u32> for RuntimeIdentifier {
+    type Error = ();
+
+    fn try_from(id: u32) -> Result<Self, Self::Error> {
+        match id {
+            0 => Ok(RuntimeIdentifier::Rust),
+            1 => Ok(RuntimeIdentifier::Java),
+            _ => Err(()),
+        }
+    }
+}
+
+impl std::fmt::Display for RuntimeIdentifier {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            RuntimeIdentifier::Rust => write!(f, "Rust"),
```

Comment:
Nit: you can use [`f.write_str`](https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.write_str).